cmd/tellstone/audit_decrypt.go

```
@@ -0,0 +1,189 @@
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
+		os.Exit(1)
+	}
+
+	// Write output.
+	if *output != "" {
+		if err := os.WriteFile(*output, out, 0o600); err != nil {
+			fmt.Fprintf(os.Stderr, "error: write output: %v\n", err)
+			os.Exit(1)
+		}
+	} else {
+		os.Stdout.Write(out)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Stdout.Write` is not checked

(errcheck)